✨Ability to pause/unpause giveaways (#269)
Androz2091/discord-giveaways: README.md

});
```

### Pause a giveaway

```js
client.on('message', (message) => {
    const args = message.content.slice(settings.prefix.length).trim().split(/ +/g);
    const command = args.shift().toLowerCase();

    if (command === 'pause') {
        const messageID = args[0];
        client.giveawaysManager.pause(messageID).then(() => {
            message.channel.send('Success! Giveaway paused!');
        }).catch((err) => {
            message.channel.send(`An error has occurred, please check and try again.\n\`${err}\``);
        });
    }
});
```

-   **options.content**: the text of the embed when the giveaway is paused.
-   **options.unPauseAfter**: the number of milliseconds after which the giveaway will automatically unpause.
-   **options.embedColor**: the color of the embed when the giveaway is paused.

> ⚠️ **Note**: the pause function overwrites/edits the [pauseOptions object property](https://github.com/Androz2091/discord-giveaways#pause-options) of the giveaway!

Reviewer: ```suggestion
⚠️ **Note**: the pause function overwrites/edits the [pauseOptions object property](https://github.com/Androz2091/discord-giveaways#pause-options) of a giveaway!
```

    <img src="https://zupimages.net/up/21/08/50mx.png"/>
</a>

### Pause a Giveaway

```js
client.on('message', (message) => {
    const args = message.content.slice(settings.prefix.length).trim().split(/ +/g);
    const command = args.shift().toLowerCase();

    if (command === 'pause') {
        const messageID = args[0];
        client.giveawaysManager.pause(messageID, {
            content: '⚠️ **THIS GIVEAWAY IS PAUSED !** ⚠️',
            unPauseAfter: null,
            embedColor: '#FFFF00'
        }).then(() => {
            message.channel.send('Success! Giveaway paused!');
        }).catch((err) => {
            message.channel.send('Encountered an error while pausing giveaway with ID ' + messageID + 'Error: ' + err + '');
        });
    }
});
```

> ### Un Pause a Giveaway

Reviewer 2: Better if just a single word.
```suggestion
### Unpause a Giveaway
```

```js
client.on('message', (message) => {
    const args = message.content.slice(settings.prefix.length).trim().split(/ +/g);
    const command = args.shift().toLowerCase();

    if (command === 'unpause') {
        const messageID = args[0];
        client.giveawaysManager.unpause(messageID).then(() => {

>             message.channel.send('Success! Giveaway un paused!');

Reviewer 2: Same as the previous suggestion.
```suggestion
            message.channel.send('Success! Giveaway unpaused!');
```

        }).catch((err) => {

>             message.channel.send('Encountered an error while un pausing giveaway with ID ' + messageID + 'Error: ' + err + '');

Reviewer 2: Same as the previous suggestion.
```suggestion
            message.channel.send('Encountered an error while unpausing giveaway with ID ' + messageID + '\nError: ' + err + '');
```

        });
    }
});
```

> 

Reviewer: I think it would be better if you move this directly under the "end" command example, since that are also cmds.
